### PR TITLE
Add fire tv specific key codes

### PR DESCRIPTION
**Pull request by TheZoker**

### CODE REVIEW

**Comment by JeffLIrion** [comment]
Can you please change `FIRE_TV` to `FIRETV`. Other than that, it looks good!

**Comment by TheZoker** [comment]
Done!

File: androidtv/constants.py [hunk 1]
```diff
@@ -163,6 +168,13 @@
         "VOLUME_UP": KEY_VOLUME_UP,
         "YELLOW": KEY_YELLOW}
 
+# Fire TV keys
+FIRETV_KEYS = KEYS
```
Comment (JeffLIrion) [inline androidtv/constants.py]: `FIRETV_KEYS = KEYS.copy()`

Comment (TheZoker) [reply androidtv/constants.py]: Fixed

**Comment by JeffLIrion** [comment]
Do any of the `media_*` methods need to be updated?

**Comment by TheZoker** [comment]
- The `media_player.media_play_pause` service does work with the current `KEY_PLAY_PAUSE` so no need to change this. 
- `rewind` and `fast-forward` are not the same as `media_player.media_next_track` and `media_player.media_previous_track`. So that's fine too
- The only thing that behaves not as expected on the Fire TV is the `media_player.media_stop` service. Somehow this just emulates the `KEY_BACK` code. But I guess that this is how amazon implemented the `KEY_STOP` code.

The only thing that is missing now is, to make the `fast-forward` and `rewind` adb command available within HA. So we need to add `FIRETV_KEYS` [here](https://github.com/JeffLIrion/home-assistant/blob/37012457dfdc2ed8a107a54e977b8f8fb9705ddd/homeassistant/components/androidtv/media_player.py#L219) and load the `key` variable differently for android tv and fire tv [here](https://github.com/JeffLIrion/home-assistant/blob/37012457dfdc2ed8a107a54e977b8f8fb9705ddd/homeassistant/components/androidtv/media_player.py#L319)

**Comment by JeffLIrion** [comment]
I intentionally used `KEY_BACK` instead of `KEY_STOP` because all `KEY_STOP` did was pause the video and I found that `KEY_BACK` behaved more like what I would expect from a stop command. 

Please make sure that the fast-forward and rewind commands actually work. You can use the command `input keyevent 228`, for example, to test fast forwarding using the `androidtv.adb_command` service.

**Comment by TheZoker** [comment]
Ok so they do not seem to work. Maybe Amazon is blocking them, when they don't come from the remote or something. I think this PR can then be closed, since these key events are useless then. 

What worked where the keys [89 ](https://developer.android.com/reference/android/view/KeyEvent.html#KEYCODE_MEDIA_REWIND) and [90](https://developer.android.com/reference/android/view/KeyEvent.html#KEYCODE_MEDIA_FAST_FORWARD)

I will create a new PR adding these two, since they are supported by android tv as well

**Comment by JeffLIrion** [comment]
Rewind and fast-forward aren't supported media player features in Home Assistant, correct? I don't see them here: [`media_player/__init__.py`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/__init__.py). 

So this would just enable the user to send the command `FAST_FORWARD` instead of `input keyevent 228`, right?

**Comment by TheZoker** [comment]
Yes exactly

**Pull request closed without merging**